Softeq.XToolkit.Common/Collections/ObservableKeyGroupsCollection.cs

```diff
@@ -697,6 +700,29 @@ private void DecrementIndex<T>(IList<KeyValuePair<T, int>> indexes)
             }
         }
 
+        private void NotifyCountIfNeeded(NotifyKeyGroupCollectionChangedEventArgs<TKey, TValue> args)
+        {
+            var countOfGroupsChanged = IsActionCanModifyGroup(args.Action);
```

Comment:
This is probably an incorrect name for a boolean variable

Reply:
Thanks, fixed